runtime/vm/classsupport.c

```diff
@@ -301,6 +301,11 @@ internalFindClassString(J9VMThread* currentThread, j9object_t moduleName, j9obje
 		UDATA utf8Length = 0;
 		PORT_ACCESS_FROM_JAVAVM(vm);
 
+		/* Make sure the name is legal */
+		if (CLASSNAME_INVALID == verifyQualifiedName(currentThread, className, expectedReturnCode)) {
+			return NULL;
```

Comment:
Please either use a goto or nest the other code inside the if here to avoid multiple return.

Reply:
Nested the following code inside this `if` block as suggested above.